nautobot/dcim/views.py

```diff
@@ -3093,7 +3093,8 @@ def render_label_wrapper(self, context):
                         table_title="Wireless Networks",
                         table_class=BaseControllerManagedDeviceGroupWirelessNetworkAssignmentTable,
                         table_attribute="wireless_network_assignments",
-                        related_field_name="controller_device_redundancy_group",
+                        related_field_name="controller_managed_device_groups__devices",
+                        related_list_url_name="wireless:wirelessnetwork_list",
```

Comment:
How were these identified as needing changes? Is there an automated test we can add?

Reply:
Django tables2 deprecation warnings.